### ISSUE TITLE
Twitter issue still not fixed

### ISSUE THREAD

**Issue by rakitanc** [libredirect/libredirect#428]
Using chrome Version 103.0.5060.114 (Official Build) (64-bit)

On Linux - Pop Os

https://github.com/libredirect/libredirect/issues/339

**Comment by EdwardLangdon** [comment]
Use The Latest Chrome Version `Chromium: 104.0.5112.102` and Addon Version 2.2.1. Then report if it doesn't work. We are also confused as other users are also reporting this in matrix.

**Comment by rakitanc** [comment]
Yeah sure..Let see if it gets fixed on my end or not

**Comment by EdwardLangdon** [comment]
@rakitanc Can you verify if your issue is fixed in v2.3 ?

**Comment by rakitanc** [comment]
> @rakitanc Can you verify if your issue is fixed in v2.3 ?

I just checked..for me it's not redirecting instagram and twitter at all

**Comment by rakitanc** [comment]
Also just none of the quora link is working too as all show offline instances

I am using this chrome version

Version 103.0.5060.114 (Official Build) (64-bit)

**Comment by rakitanc** [comment]
Just an update some links did redirect randomly from google searches but one thing for sure that going on instagram.com manually not redirecting that at all

**Comment by EdwardLangdon** [comment]
Can you download the zip file from here : 

![image](https://user-images.githubusercontent.com/101938856/195966207-478f5f3b-4851-4b8b-a5a1-fb275d88bf11.png)

Extract the zip.

Go to Extensions in your Browser. Enable developer mode and choose `Load Unpacked`. Navigate to the extracted folder. Choose `src` folder.



> I am using this chrome version
> 
> Version 103.0.5060.114 (Official Build) (64-bit)

🤯 , What !?

The Latest Chromium version is 106.0.5249.119. You are very lagging behind with unpatched vulnerabilites risking yourself.

Check for updates for Pop Shop. You should enable Auto updates or check for updates regularly.

**Comment by rakitanc** [comment]
Yeah man..The reason i am lagging behind because before that i need to update my OS as well as the version i am currently on the POP os team has stopped the support of it and now to install the latest update from what i read i have to do a complete refresh install which kinda makes me nervous because then i have to all the setup on chrome and settings on PC all over again which i have done overtime.

Can you help me to make it easier to backup and install?

**Comment by EdwardLangdon** [comment]
> Just an update some links did redirect randomly from google searches but one thing for sure that going on instagram.com manually not redirecting that at all

https://cadence.moe/blog/2022-09-01-discontinuing-bibliogram

**Comment by rakitanc** [comment]
So what to do now mate?

**Comment by EdwardLangdon** [comment]
We now redirect only Instagram posts not account profiles.

The Question I ask is, Is your main issue fixed ?
